export-image/00-allow-rerun/00-run.sh

```diff
@@ -3,7 +3,3 @@
 if [ ! -x "${ROOTFS_DIR}/usr/bin/qemu-aarch64-static" ]; then
 	cp /usr/bin/qemu-aarch64-static "${ROOTFS_DIR}/usr/bin/"
 fi
-
-if [ -e "${ROOTFS_DIR}/etc/ld.so.preload" ]; then
-	mv "${ROOTFS_DIR}/etc/ld.so.preload" "${ROOTFS_DIR}/etc/ld.so.preload.disabled"
-fi
```

Comment:
This file was only created on the arm32 version of Umbrel OS because RPi used it to overwrite some built-in function with one optimized for their CPU.